pandas/core/frame.py

```diff
@@ -5733,6 +5740,12 @@ def reset_index(
         allow_duplicates : bool, optional, default lib.no_default
             Allow duplicate column labels to be created.
 
+            .. versionadded:: 1.5.0
+        names : int, str or 1-dimensional list, default None
```

Comment:
i think you need a blank line *after* the versionadded

Reply:
Done